### PR TITLE
Closes #001 - Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

**Review by tarasbiletsky24 — CHANGES_REQUESTED** [review]
Add empty line at the end of the file

**Review by nikita-namely — CHANGES_REQUESTED** [review]
Add empty line in the end of file

**Review by martakondor — CHANGES_REQUESTED** [review]
Add empty line in the end of the file.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (bibipkins) [inline index.html]: Change file ending. It must end with empty row

Comment (stasoz) [inline index.html]: Add a newline at end of file

**Review by vov4ik-7 — CHANGES_REQUESTED** [review]
It's necessary to add an empty line at the end of the file.

**Review by tarasbiletsky24 — APPROVED** [review]

**Review by stasoz — APPROVED** [review]

**Review by vov4ik-7 — APPROVED** [review]

**Review by martakondor — APPROVED** [review]

**Review by nikita-namely — APPROVED** [review]

**Comment by koldovsky** [comment]
Closes 001 - Create initial index.html

**Pull request merged**